fs/test/test_exportfs.cpp

```diff
@@ -104,6 +108,8 @@ int callbackvoid(void*, AsyncResult<void>* ret) {
     }
 
 TEST(ExportFS, basic) {
+    photon_init();
```

Comment:
why not doing init in main()?

Reply:
Other tests like ExportFS.no_xattr_sync run in pthread rather than photon thread.